model-optimizer/extensions/front/tf/KerasRNNReplacer.py

```diff
@@ -0,0 +1,237 @@
+"""
```

Comment:
nit: avoid usage of word `Replacer` in favour of `Transformation`

Reply:
Fixed.